ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -55,14 +59,16 @@ const getSteps = ({
                 <Title headingLevel="h2" size="xl">
                     {label}
                 </Title>
-                {stepNotification &&
-                 (stepNotification.step === currentStepId) &&
-                 <Alert
-                   isInline
-                   title={stepNotification.message}
-                   variant="danger"
-                 />}
-                {children}
+                <TextContent>
+                    {stepNotification &&
+                     (stepNotification.step === currentStepId) &&
+                     <Alert
+                       isInline
+                       title={stepNotification.message}
+                       variant="danger"
+                     />}
+                    {children}
+                </TextContent>
```

Comment:
I believe you should have instead wrapped only the `Alert` in `TextContent` and kept the logic outside, but it probably works this way too. I'm concerned mainly because of any side effects of the empty `TextContent` that is now always present, but that's because I don't know what that will really do.

To clarify:
- old was `{logic && <Alert>...`
- you made it `<TextContent>{logic && <Alert>...`
- I suggest `{logic && <TextContent><Alert>...`

Reply:
Actually, the title should be there as well. It is not about the alert, but the whole content of the wizard step (the right part). The `TextContent` component is just a simple `div` that will fix the style of the text on that page.